ext/pdo_mysql/tests/pdo_mysql_dsn_containing_credentials.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+PDO_MYSQL: Pass credentials in dsn instead of constructor params
+--SKIPIF--
+<?php
+require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'skipif.inc');
+require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
+MySQLPDOTest::skip();
+?>
+--FILE--
+<?php
+	require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
```

Comment:
Actually, this seems to be required; without it the build fails, see [this CI run](https://ci.appveyor.com/project/php/php-src/build/master.build.3182/job/psfgcfagao6lo06j#L4986) (fails with `Class 'PDOTest' not found`)